src/firefly/js/tables/ui/BasicTableView.jsx

```diff
@@ -29,7 +29,7 @@ export const BY_SCROLL = 'byScroll';
 const BasicTableViewInternal = React.memo((props) => {
 
     const {width, height} = props.size;
-    const {columns, data, hlRowIdx, renderers, bgColor, selectInfoCls, callbacks, rowHeight,
+    const {columns, data, hlRowIdx, renderers, bgColor, selectInfoCls, callbacks, rowHeight, rowHeightGetter, showHeader=true,
```

Comment:
Can you also include the added option `showHeader` to tables-typedefs.jsdoc and table-api.html?  I like to keep them in sync.